packages/docs/content/migration-guides/introducing-a-css-reset.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Introducing a CSS reset
+order: 10
+---
+
+The design system's CSS reset is a collection of HTML element-specific styles that simplifies the development process while providing cross-browser sensible defaults to build from.
+
+Our reset CSS is bundled with the rest of the design system's styles, so to use it all you need to do is [import the design system's CSS](/getting-started/for-developers/#including-the-css) using your preferred method. Even if you haven't imported one of our components, your application should visually align with the design system.
+
+If you're curious to see what changes have been made, you can preview our reset styles in [Storybook](/storybook/?path=/story/reset-overview--overview). We'll break down the major changes for you to review below.
+
+## Global defaults
+
+The CSS reset introduces globally-defined defaults that will impact the entire application once the CSS. These defaults include (but aren't limited to):
+
+- `box-sizing: border-box` is set on every element — including `*::before` and `*::after`. This ensures that the declared width of element is never exceeded due to padding or border.
+- The `<html>` element has `scroll-behavior: smooth` set on `:focus-within`. This is a CSS-only solution that provides smooth anchor scrolling.
+- Animations, transitions, and smooth scroll are removed on all elements - including `*::before` and `*::after` for users who have `prefers-reduced-motion` enabled to respect their preferences.
+- The `<body>` element effectively replaces the `.ds-base` class, defining `font-family`, `font-size`, `line-height`, and `color`.
+  - In addition, `<body>` now sets `background-color` to a theme's specified background.
+- The `outline` is removed on elements with the `tabindex="-1"` declared. This is removes focus rings for non-interactive elements, e.g., when activating a skip link will then send focus to a `<main>` or heading element.
+- To avoid [margin collapse](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing), all elements will only rely on their respective top margins to better control layout. All bottom margins have been removed.
+
+## Typography
+
+### Headings
+
+All heading elements will receive the font styles defined by their parent element using the `font: inherit` rule. 
+
+[`font`](https://developer.mozilla.org/en-US/docs/Web/CSS/font) is a shorthand property that encompasses several font styles, but the key thing to note is a heading will now inherit the `font-family` and `font-size` of its parent or `<body>` element. This rule ensures the semantics of a heading element are distinct from its appearance. If you need to style a heading, refer to our [headings](/foundation/typography/headings/) or [utility class](/utilities/overview/) documentation.
+
+Lastly, headings have been reset to have their bottom margins removed. All headings have a top margin set to `1.5em`.
```

Comment:
Should we explain why it's `1.5` instead of on the 8px grid? Because it's relative to the heading's font size?

Reply:
I'm not sure if we need to explain why it's `1.5em`... part of me wonders if this line should be removed entirely. I do specify on line 22 that we're pivoting to using top margins only and maybe that's enough? By hard-coding a value here, does this run the risk of falling out of date with the codebase as we go? And does this margin apply to all themes? I think including this info raises more questions than it solves.

Reply:
Yeah, that's a good point. I think it's nice to at least call out that it has top margins and not bottom margins